pkgs/development/python-modules/mkdocs/default.nix

```diff
@@ -0,0 +1,84 @@
+{ lib
+, python3
+, fetchFromGitHub
+, buildPythonPackage
+, fetchpatch
+, click
+, jinja2
+, markdown
+, mergedeep
+, pyyaml
+, pyyaml-env-tag
+, ghp-import
+, importlib-metadata
+, watchdog
+, packaging
+, Babel
+, mock
+, pytestCheckHook
+, pythonOlder
+}:
```

Comment:
```suggestion
# build time
, foo

# runtime
, bar

# tests
, baz
}:
```

It is open helpful to sort these inputs by their purpose, especially when the list gets longer.

Reply:
Roger that.